node/src/reactor/participating/config.rs

```diff
@@ -38,4 +39,7 @@ pub struct Config {
     pub deploy_acceptor: DeployAcceptorConfig,
     /// Linear chain sync configuration.
     pub linear_chain_sync: LinearChainSyncConfig,
+    /// Block proposer configuration.
+    #[serde(default)]
```

Comment:
Is this to make this configuration optional? I can see why that is useful, but on the other hand, wouldn't that be slightly out of policy regarding the rest of our code?

Reply:
Yes, I noticed that too.  This was done to keep it as close to the original code in release-1.2 as possible, to help simplify the original PR's review.  Again, this should be getting removed in the very near future, so I wasn't too concerned.